curriculum/getChallenges.js

```diff
@@ -178,24 +178,18 @@ async function buildBlocks({ basename: blockName }, curriculum, superBlock) {
     `./challenges/_meta/${blockName}/meta.json`
   );
   let blockMeta;
-  try {
-    blockMeta = require(metaPath);
+
+  if (fs.existsSync(metaPath)) {
+    blockMeta = JSON.parse(fs.readFileSync(metaPath, 'utf8'));
+
     const { isUpcomingChange } = blockMeta;
-    if (typeof isUpcomingChange !== 'boolean') {
-      throw Error(
-        `meta file at ${metaPath} is missing 'isUpcomingChange', it must be 'true' or 'false'`
-      );
-    }
 
     if (!isUpcomingChange || showUpcomingChanges) {
       // add the block to the superBlock
       const blockInfo = { meta: blockMeta, challenges: [] };
       curriculum[superBlock].blocks[blockName] = blockInfo;
     }
-  } catch (e) {
-    if (e.code !== 'MODULE_NOT_FOUND') {
```

Comment:
Now, we do not have this check. Perhaps this needs an update, but we should have all IO ops in a `try...catch`